packages/jupyter-kernel/jupyter-kernel.0.3/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: ["Simon Cruanes" "Andrew Ray"]
+homepage: "https://github.com/ocaml-jupyter/jupyter-kernel"
+bug-reports: "https://github.com/ocaml-jupyter/jupyter-kernel/issues"
+tags: ["jupyter" "ipython"]
+dev-repo: "https://github.com/ocaml-jupyter/jupyter-kernel.git"
+build: [make "build"]
+build-doc: ["jbuilder" "doc"]
+depends: [
+  "jbuilder" {build}
+  "base-bytes"
+  "result"
+  "base-unix"
+  "zmq"
+  "atdgen" {build}
```

Comment:
It's a bit surprising that this is only a build dependency. Keep in mind that atdgen produces files that require a runtime (`atdgen.runtime`)`. I'm not sure how you managed to avoid it.